android/src/main/java/com/stripeterminalreactnative/ktx/TerminalExtensions.kt

```diff
@@ -98,49 +45,45 @@ suspend fun Terminal.connectReader(
     reader: Reader,
     locationId: String,
     autoReconnectOnUnexpectedDisconnect: Boolean = false,
-    listener: ReaderListenable? = null,
-    reconnectionListener: ReaderReconnectionListener
+    disconnectListener: ReaderDisconnectListener? = null,
+    reconnectionListener: ReaderReconnectionListener,
```

Comment:
Is this `connectReader(...)` method publicly exposed in the RN SDK? If not, can we remove it?